index.html

```diff
@@ -11,34 +11,217 @@
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#" class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#"class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                    </div>  
+                </div>
+            </div>
+        </div>
+        <section class="about-container">
+            <div class="first-about-container">
+                <h1 class="about-header">About me</h1>
+                <p class="about-para">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#"><button class="about-btn-text hover-effect">Get My Resume</button></a>
+                <img class="about-illu" src="./img/about/illustration-about-me-mobile.svg" alt="illustration">
+            </div>
+            <img class="about-divider" src="./img/about/Vector 4.svg" alt="">
```

Comment:
Please consider following the design closely. In design the image can be seen after the background-image for the design but in your case its not the same. Please consider reviewing your design and making it in accordance with the template.

# Difference

## Figma Design

![image](https://user-images.githubusercontent.com/94223367/220149801-6db28abc-2d49-4f4d-96f8-109135fe3a32.png)


## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220149846-2f45f017-8666-4207-b479-4fd11daf3234.png)